packages/web-components/fast-ssr/src/dom-shim.ts

```diff
@@ -0,0 +1,24 @@
+import { installWindowOnGlobal } from "@lit-labs/ssr/lib/dom-shim.js";
+
+installWindowOnGlobal();
+
+// Implement shadowRoot getter on HTMLElement.
+// Can be removed if https://github.com/lit/lit/issues/2652
+// is integrated.
+class PatchedHTMLElement extends HTMLElement {
```

Comment:
Should we contribute this back to Lit? Or should we have our own DOM shim? Maybe something like open web components should own a few of these things in a package that's independent of any library?

Reply:
I actually already contributed it back to lit, just waiting on publishing.